### PR TITLE
feat: add app

### PR DESCRIPTION
**Pull request by davenportjw**
- app and terraform updates

### CODE REVIEW

File: src/retrieval_service/cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Kaniko: dockerfile location and context dir
+# General build step: dir: arg for step
+- name: 'gcr.io/kaniko-project/executor:latest'
+  args:
+  - --dockerfile=Dockerfile
+  - --destination=us-central1-docker.pkg.dev/$PROJECT_ID/cloud-run-source-deploy/retrieval-service
+  - --cache=true
+  - --cache-ttl=6h
+# # Deploy container image to Cloud Run
+# - name: 'gcr.io/google.com/cloudsdktool/cloud-sdk'
```
Comment (kweinmeister) [inline src/retrieval_service/cloudbuild.yaml:26]: Is this supposed to be commented out?

File: src/retrieval_service/datastore/providers/firestore_test.py [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+    def collection(self, collection_name: str):
+        return self.collections[collection_name]
+
+
+async def mock_client(mock_firestore_client: MockFirestoreClient) -> firestore.Client:
+    return firestore_provider.Client(mock_firestore_client)
+
+
+async def test_get_airport_by_id():
```
Comment (kweinmeister) [inline src/retrieval_service/datastore/providers/firestore_test.py:78]: Consider a negative test or two - not found or invalid inputs

File: src/retrieval_service/run_database_export.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+async def main():
+    cfg = parse_config()
+    ds = await datastore.create(cfg.datastore)
+
+    airports, amenities, flights = await ds.export_data()
+
+    await ds.close()
+
+    airports_new_path = "../data/airport_dataset.csv.new"
```
Comment (kweinmeister) [inline src/retrieval_service/run_database_export.py:32]: What are these new files? Consider a comment if these are correct

File: test/integration/simple_example/simple_example_test.go [hunk 4]
```diff
@@ -38,56 +35,9 @@ func TestSimpleExample(t *testing.T) {
 	dwh.DefineVerify(func(assert *assert.Assertions) {
 		dwh.DefaultVerify(assert)
 
-		projectID := dwh.GetTFSetupStringOutput("project_id")
+		// projectID := dwh.GetTFSetupStringOutput("project_id")
+		// TODO add tests
```
Comment (kweinmeister) [inline test/integration/simple_example/simple_example_test.go:39]: Looks like these will get added later

**Review by kweinmeister — APPROVED** [review]

File: database.tf [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    }
+  }
+  deletion_protection = var.deletion_protection
+
+  depends_on = [
+    google_service_networking_connection.main,
+  ]
+}
+
+# # Create Database
```
[the remark this answers is not in the document]
Comment (davenportjw) [reply database.tf:79]: I did that to group the objects

File: src/frontend_service/run_app.py [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import uvicorn
+
+from main import init_app
+
+
+async def main():
+    PORT = int(os.getenv("PORT", default=8080))
+    HOST = os.getenv("HOST", default="0.0.0.0")
+    CLIENT_ID = os.getenv("CLIENT_ID")
+    SECRET_KEY = os.getenv("SECRET_KEY")
```
[the remark this answers is not in the document]
Comment (davenportjw) [reply src/frontend_service/run_app.py:28]: These are holdovers on the other codeset that I'll need to sync with teh SWEs to change, it's not used in this context.

**Pull request merged**